src/main/java/edu/cornell/kfs/concur/service/impl/ConcurAccessTokenServiceImpl.java

```diff
@@ -68,6 +115,94 @@ protected ClientRequest buildRefreshAccessTokenClientRequest() {
     }
 
     @Override
+    public void revokeAccessToken() {
```

Comment:
Thanks for the suggestions. I agree that we don't need the revoke-only functionality, and that having a revoke-and-request-new action would be better.

Because of our current environment setups, though, we may need some way to request a new token without revoking the current one. Otherwise, we could run into issues because DEV and TEST are currently using the same token. Perhaps an appropriate question prompt can be used to give the user more control about whether to revoke the token before requesting a new one, or perhaps we need to redo our process for bootstrapping tokens in DEV and TEST.